action.yml

```diff
@@ -17,6 +17,10 @@ inputs:
   features:
     description: Comma-separated list of cargo build features to enable
     required: false
+  package:
+    description: Whether to build with `--package` option
+    required: false
+    default: 'none'
```

Comment:
`none` is a valid package name, so a empty string is probably better default value.

https://crates.io/crates/none